src/widget/wpushbutton.h

```diff
@@ -106,12 +106,24 @@ class WPushButton : public WWidget {
     // Associated background pixmap
     PaintablePointer m_pPixmapBack;
 
-    QPixmap m_preLongPressPixmap;
-
     // short click toggle button long click push button
     ControlPushButton::ButtonMode m_leftButtonMode;
     ControlPushButton::ButtonMode m_rightButtonMode;
     QTimer m_clickTimer;
-    QTimer m_animTimer; // To animate long press latching
     QVector<int> m_align;
+
+    class LongPressLatching {
```

Comment:
please add an explanatory comment for this new class that summarizes the feature

Reply:
Will do.

Reply:
Comment added. If you find it's sufficient, could you merge this? Thanks!

Reply:
@daschuer is the last outstanding reviewer